simple-stack-example-kotlin-dagger/src/main/java/com/zhuinden/simplestackkotlindaggerexample/userdetail/UserDetailFragment.kt

```diff
@@ -0,0 +1,53 @@
+package com.zhuinden.simplestackkotlindaggerexample.userdetail
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.zhuinden.simplestackkotlindaggerexample.BaseFragment
+import com.zhuinden.simplestackkotlindaggerexample.R
+import com.zhuinden.simplestackkotlindaggerexample.realmobjects.UserRO
+import com.zhuinden.simplestackkotlindaggerexample.requireArguments
+import kotlinx.android.synthetic.main.fragment_user_detail.userEmail
+import kotlinx.android.synthetic.main.fragment_user_detail.userName
+import kotlinx.android.synthetic.main.fragment_user_detail.userPhoneNumber
+
+
+class UserDetailFragment : BaseFragment() {
+
+    private lateinit var userRO: UserRO
+
+    companion object {
+        private const val ARG_USER_RO = "userRO"
+        @JvmStatic
+        fun newInstance(userRO: UserRO) =
+            UserDetailFragment().apply {
+                arguments = Bundle().apply {
```

Comment:
Now that you mention it, why isn't `UserRO` just a key argument? Then you can access `UserRO` from args via `getKey<UserDetailKey>().userRO` which is typed